WcaOnRails/app/models/competition.rb

```diff
@@ -1642,6 +1642,10 @@ def competition_series_wcif(authorized: false)
     competition_series&.to_wcif(authorized: authorized)
   end
 
+  def competition_series_ids
+    competition_series&.competition_ids
```

Comment:
Not a blocker, just a genuine question: Would it make sense to return something like `competition_series&.competition_ids || []` to make sure it's always an array?

Reply:
would basically save a null check on the other side

Reply:
Oooh yes I like that a lot more, will make the change!

Reply:
@dunkOnIT Kindly ping :)

Reply:
I pushed to it as Duncan is currently traveling and I was doing monolith work anyway

Reply:
Thanks Finn! And sorry yeah. it's been a juggling act with the travelling :P